tools/sanity_checks.py

```diff
@@ -300,6 +300,10 @@ def check_source_url(self, name: str, wrap_section: configparser.SectionProxy, v
         elif name == 'directxmath':
             # DirectXMath source url contains only tag name without version
             return True
+        elif name == 'x-plane-sdk':
+            segs = version.split('.')
+            assert(len(segs) == 3)
```

Comment:
Any reason to use a bare assert here?

Note that a bare assert should NOT use parentheses, as it is not a function. Spell this as:

```python
assert len(segs) == 3
```

The extra parentheses are interpreted as
```python
to_check = (len(segs) == 3)

assert to_check
```

Reply:
Ok after taking another look I notice this is copied from sqlite which does the same broken thing a few lines up... sigh...

Reply:
I'm innocent, I promise!